doc/classes/File.xml

```diff
@@ -401,6 +401,7 @@
 			<description>
 				Stores the given [String] as a line in the file.
 				Text will be encoded as UTF-8.
+				This is like [method store_string] but appends a newline.
```

Comment:
Suggestion (rewriting the whole docstring):

> Appends a line return character followed by [code]line[/code] to the file, encoding the text as UTF-8.

Reply:
Note that it's the other way around (`line + '\n'`).

> Appends [code]line[/code] to the file followed by a line return character ([code]\n[/code]), encoding the text as UTF-8.